### ISSUE TITLE
can i load map form my local disk？

### ISSUE THREAD

**Issue by w8128835** [swri-robotics/mapviz#744]
hello, due to the speed of network, the tile maps loaded by url are too slow, especially being maps. could you please tell me how to load tile maps from my local disk?

**Comment by danthony06** [comment]
Yes. We have not documented the procedure extensively, but this issue has an example of how to do it: https://github.com/swri-robotics/mapviz/issues/575.

**Comment by danthony06** [comment]
Closing as stale.
